src/System.IO.Pipelines/src/Resources/Strings.resx

```diff
@@ -134,6 +135,12 @@
   <data name="GetResultBeforeCompleted" xml:space="preserve">
     <value>Can't GetResult unless awaiter is completed.</value>
   </data>
+  <data name="InvalidExaminedOrConsumedPosition" xml:space="preserve">
+    <value>The examined position must be greater than or equal to the consumed position.</value>
+  </data>
+  <data name="InvalidExaminedPosition" xml:space="preserve">
+    <value>Invalid examined position specified. The current examined position is less than the current examined position.</value>
```

Comment:
`The current examined position is less than the current examined position.`
Has two references to `current examined`.

Reply:
Man typo hell, nice catch